debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,15 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # This is a somewhat of a blank check for mount profiles.  Mount profiles
+    # are written by the trusted snapd to a root-writable location and
+    # processed by snap-confine. Regardless of configuration options and
+    # runtime factors they are processed after pivot_root (on classic systems)
+    # so snap.rootfs is not a factor.
+    mount options=(rw bind) /** -> /**,
```

Comment:
First, this rule could be rewritten as: `mount options=(rw bind),`

Second, I don't like this rule. Because snap-confine is setuid we are trying to be fanatical about it's profile which is why we don't use the base abstraction. I prefer updating the profile with each newly added mount point. Is there a reason why we can't continue to do this? If not, is there a way to limit this to /snap/ubuntu-core/** or something?